.buildkite/hooks/post-command

```diff
@@ -2,6 +2,8 @@
 
 if [[ "$BUILDKITE_AGENT_NAME" =~ ^bk-agent ]]; then
   echo "Pipeline file triggered from outside the kibana executors, skipping post_command"
+elif [[ "$SKIP_KIBANA_HOOKS" =~ ^(1|true)$ ]]; then
```

Comment:
In some cases, when we want to run a command that's not kibana-related (e.g.: `buildkite-agent annotate`, or any shell command outside kibana), then we don't need to clone the kibana repo, and set up yarn/etc. With this we can shave off a few seconds per step